### PR TITLE
ENT-3486 | Added the subscribe functionality for code assignment nudg…

### PR DESCRIPTION
**Pull request by hasnain-naveed**
…e emails.

### CODE REVIEW

File: ecommerce/extensions/api/serializers.py [hunk 1]
```diff
@@ -1383,6 +1384,7 @@ class CouponCodeAssignmentSerializer(serializers.Serializer):  # pylint: disable
         child=OfferAssignmentSerializer(), read_only=True
     )
     base_enterprise_url = serializers.URLField(required=False, write_only=True)
+    email_auto_reminder = serializers.BooleanField(required=False)
```
Comment (saleem-latif) [inline ecommerce/extensions/api/serializers.py]: nit: I think a different name of something like `enable_email_reminders` would be better?

Comment (saleem-latif) [reply ecommerce/extensions/api/serializers.py]: or simply `enable_nudge_emails`

**Review by saleem-latif — APPROVED** [review]

File: ecommerce/extensions/api/serializers.py (hunk 1 above)
Comment (hasnain-naveed) [reply ecommerce/extensions/api/serializers.py]: @muhammad-ammar

Comment (saleem-latif) [reply ecommerce/extensions/api/serializers.py]: Also, instead of `required=False`, wouldn't it better to have `default=False`

**Review by saleem-latif — APPROVED** [review]

**Review by muhammad-ammar — APPROVED** [review]
LGTM

**Review by irfanuddinahmad — APPROVED** [review]
LGTM!

**Pull request merged**